action.yml

```diff
@@ -4,6 +4,8 @@ author: 'GitHub'
 inputs:
   repo-token:
     description: 'The GITHUB_TOKEN secret'
+    required: false
+    default: ${{ github.token }}
```

Comment:
What would you expect?
As it's still the same thing that needs to be passed, it just has a default value now.

Reply:
I think the description was created incorrectly initially. The description does not explain the purpose of use and it is not necessary to use only the `GITHUB_TOKEN`, it can also be a personal token. I think the following information will be more appropriate: `The GitHub token used to manage labels`